dags/multipod/maxtext_end_to_end.py

```diff
@@ -82,3 +83,108 @@
         docker_image=DockerImage.MAXTEXT_GPU_JAX_STABLE.value,
         test_owner=test_owner.NINA_C,
     ).run()
+    nightly_gpu = gke_config.get_maxtext_end_to_end_gpu_gke_test_config(
+        accelerator_type=GpuVersion.XPK_H100,
+        gpu_zone=Zone.US_CENTRAL1_C.value,
+        time_out_in_min=300,
+        test_name=f"{test_name_prefix}-nightly-{model}",
+        test_script=test_script,
+        num_slices=2,
+        cluster_name=ClusterName.A3_CLUSTER.value,
+        docker_image=DockerImage.MAXTEXT_GPU_JAX_NIGHTLY.value,
+        test_owner=test_owner.NINA_C,
+    ).run()
+    stable_tpu >> nightly_tpu >> stable_gpu >> nightly_gpu
+
+  multicluster_test_models = {
+      "gemma-7b": [
+          {
+              "script_name": "tpu/gemma/7b/1_test_gemma",
+              "cpu_device_type": CpuVersion.N2_STANDARD,
+              "cpu_zone": Zone.US_CENTRAL1_B.value,
+              "cluster_name": ClusterName.CPU_N2_STANDARD_64.value,
+          },
+          {
+              "script_name": "tpu/gemma/7b/2_test_gemma",
+              "tpu_version": TpuVersion.V4,
+              "tpu_cores": 16,
+              "cluster_name": ClusterName.V4_16_MULTISLICE_CLUSTER.value,
+              "tpu_zone": Zone.US_CENTRAL2_B.value,
+          },
+      ]
+  }
+
+  for model, test_scripts_details in multicluster_test_models.items():
+    gcs_subfolder = f"{test_owner.Team.MULTIPOD.value}/maxtext"
+
+    test_group_id = "chained_tests" + "_" + model + "_" + "stable"
+
+    with TaskGroup(group_id=test_group_id, prefix_group_id=False) as group:
```

Comment:
A high-level thought - it would be good if we can hide the implementation details like TaskGroup and `shared_gcs_location` and make these chained tests a config in `gke_config.py`, but it probably requires dedicated support. We can follow up on this on the infra side.

Reply:
SG - That would be great